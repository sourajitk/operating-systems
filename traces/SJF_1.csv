1,0,1
2,0,3
3,0,2
4,7,3
5,8,2
6,9,1
