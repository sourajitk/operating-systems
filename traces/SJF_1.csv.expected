1,1
2,6
3,3
4,10
5,13
6,11
